.dockerignore

```diff
@@ -1,6 +1,5 @@
 .dockerignore
 .editorconfig
-.travis.yml

```

Comment:
Why did you remove this if you have it at https://github.com/BeeStation/NSV13/blob/master/.dockerignore?

Reply:
Because I forgot to remove it there, as I am a fool.